src/ui/FolderContent/FolderContent.jsx

```diff
@@ -0,0 +1,83 @@
+import { getCategory } from "data-access/getCategory";
+import "./FolderContent.css";
+import { Button } from "StyledComponents/Buttons";
+import { useEffect, useState } from "react";
+import { CategoryNav } from "ui/CategoryNav/CategoryNav";
+import { CardList } from "ui/CardList";
+import { ReadOnlyCard } from "ui/ReadOnlyCard";
+import { EmptyLink } from "ui/EmptyLink/EmptyLink";
+import { getFolders, getFolder } from "../../data-access/getFolders";
+
+export function FolderContent() {
+  const [folder, setFolder] = useState([]);
+  const [data, setData] = useState();
+  const [folderId, setFolderId] = useState("");
```

Comment:
P3:
기본값이 string일 필요가 있을까요?
빈값일 때는 undefined 인게 더 코드파악하기 좋을 것 같아요.

```suggestion
  const [folderId, setFolderId] = useState();
```